wolfssl/wolfcrypt/settings.h

```diff
@@ -1621,7 +1621,7 @@ extern void uITRON4_free(void *p) ;
 #ifdef HAVE_CRL
     /* not widely supported yet */
     #undef NO_SKID
```

Comment:
I'm not sure this undef should stay in.  Silently removing potential user settings is not ideal.